DigitalLearningSolutions.Data/DataServices/UserDataService.cs

```diff
@@ -235,7 +235,10 @@ public List<DelegateUser> GetDelegateUsersByUsername(string username)
                         Surname AS LastName,
                         Email AS EmailAddress,
                         Password,
-                        ResetPasswordID
+                        ResetPasswordID,
+                        CentreID,
+                        IsCentreManager,
+                        CentreAdmin AS IsCentreAdmin
                     FROM AdminUsers
                     WHERE (Email = @emailAddress)",

```

Comment:
This is likely the cause of one of the merge conflicts. I updated this query to return a lot more values (including the ones you've added it it here).

The idea is that going forwards, we should fully populate all non-nullable properties of the object we are querying into to ease unit testing (we can just check equivalence to defined default objects from the test helpers) and it prevents unexpected errors in the future when someone else uses the method expecting non-nullable properties not to not have null values. If we really don't want to fully populate the object we should define a new object that only has the properties we want.